pkg/collector/corechecks/snmp/session.go

```diff
@@ -0,0 +1,130 @@
+package snmp
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gosnmp/gosnmp"
+)
+
+const sysObjectIDOid = "1.3.6.1.2.1.1.2.0"
+
+// Java SNMP uses 50, snmp-net uses 10
+// Same max repetition as gosnmp.defaultMaxRepetitions
+const bulkMaxRepetition = 50
```

Comment:
Does it worth having an external configuration for `bulkMaxRepetition` ?

Reply:
Both `defaultOidBatchSize = 60` and `bulkMaxRepetition = 50` are not exposed as config. 

I think it's better to keep them private with reasonable defaults unless we prove that it will bring value to users to customise them for specific circumstances. From past feedbacks, changes to `oid_batch_size` config in Python didn't led to significant performance gain/loss.